src/Perspex.Controls/LayoutTransformControl.cs

```diff
@@ -0,0 +1,407 @@
+﻿// Copyright (c) The Perspex Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+//
+// Idea got from and adapted to work in perspex
+// http://silverlight.codeplex.com/SourceControl/changeset/view/74775#Release/Silverlight4/Source/Controls.Layout.Toolkit/LayoutTransformer/LayoutTransformer.cs
+//
+
+using Perspex.Controls.Presenters;
+using Perspex.Controls.Primitives;
+using Perspex.Controls.Templates;
+using Perspex.Media;
+using Perspex.VisualTree;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Reactive.Linq;
+
+namespace Perspex.Controls
+{
+    public class LayoutTransformControl : ContentControl
+    {
+        public static readonly PerspexProperty<Transform> LayoutTransformProperty =
+            PerspexProperty.Register<LayoutTransformControl, Transform>(nameof(LayoutTransform));
+
+        static LayoutTransformControl()
+        {
+            LayoutTransformProperty.Changed
+                .AddClassHandler<LayoutTransformControl>(x => x.OnLayoutTransformChanged);
+            TemplateProperty.OverrideDefaultValue<LayoutTransformControl>(_defaultTemplate);
+        }
+
+        public Transform LayoutTransform
+        {
+            get { return GetValue(LayoutTransformProperty); }
+            set { SetValue(LayoutTransformProperty, value); }
+        }
+
+        /// <summary>
+        /// Acceptable difference between two doubles.
+        /// </summary>
+        private const double AcceptableDelta = 0.0001;
+
+        /// <summary>
+        /// Number of decimals to round the Matrix to.
+        /// </summary>
+        private const int DecimalsAfterRound = 4;
+
+        private static readonly FuncControlTemplate<LayoutTransformControl> _defaultTemplate =
```

Comment:
No other controls have a default template expressed in code, probably best to just add a XAML template to Perspex.Styles.Default as with other controls.